a10_octavia/controller/worker/tasks/vthunder_tasks.py

```diff
@@ -333,3 +339,27 @@ def execute(self, master_vrrp_status, backup_vrrp_status):
             return True
         else:
             return False
+
+
+class HandleACOSPartitionChange(task.Task):
```

Comment:
Shouldn't this be CreateACOSPartition Task ?

Reply:
It could create it or it could just change to it. We try to not use words like "Create, Update, Delete" unless they exclusively are doing that operation.

One of the ways they've handled this in octavia is by tasks names as "Handle<Object>Delta". So that's being emulated here.